modules/javafx.controls/src/main/java/javafx/scene/control/ComboBox.java

```diff
@@ -240,8 +240,9 @@ public ComboBox(ObservableList<T> items) {
         // selection model to indicate that this is the selected item
         valueProperty().addListener((ov, t, t1) -> {
             if (getItems() == null) return;
-
             SelectionModel<T> sm = getSelectionModel();
+            if (sm == null) return;
```

Comment:
Do we need similar null check in editableProperty() listener below?

Reply:
Yes. I just focused on the value but you are right, we should add it there as well. Will update my PR and also document it in the ticket.